coriolis/osmorphing/manager.py

```diff
@@ -239,6 +239,10 @@ def morph_image(origin_provider, destination_provider, connection_info,
         LOG.info("Post packages uninstall")
         export_os_morphing_tools.post_packages_uninstall(packages_remove)
 
+    LOG.info("Checking for packages already installed")
+    packages_add = import_os_morphing_tools.check_installed_packages(
```

Comment:
You can replace this with calling `get_installed_packages`, and then only install the set difference